src/api/libcellml/variable.h

```diff
@@ -39,6 +39,20 @@ class LIBCELLML_EXPORT Variable: public NamedEntity
     Variable& operator=(Variable n); /**< Assignment operator */
 
     /**
+     * @brief The INTERFACE_TYPES enum.
+     * Enum describing the interface types that are available for
+     * a given Variable.
+     */
+    enum INTERFACE_TYPES
+    {
+        INTERFACE_TYPE_UNDEFINED,
```

Comment:
Thanks @agarny , the undefined type is there to discern between an interface set to `none` and one without any interface specified. According to the 1.2 spec, a given variable doesn't necessarily have an interface.